tasks/src/image-segmentation/about.md

```diff
@@ -0,0 +1,38 @@
+## Use Cases
+The most important use of image segmentation is in computer vision for robotics, such as autonomous driving. Segmentation models are used to identify pedestrians, lanes and other necessary information. Image segmentation models are also used in cameras to erase background of portraits in images.
+
+## Task Variants
+### Semantic Segmentation 
```

Comment:
Maybe you can add the images you did for the presentation that clearly show the difference between the 3 types